doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -519,7 +519,7 @@ but moderate. The minimum timeouts should never be below 10 seconds.
 Location to store the resource state in.
 </longdesc>
 <shortdesc lang="en">State file</shortdesc>
-<content type="string" default="/var/run//Dummy-{OCF_RESOURCE_INSTANCE}.state" />
+<content type="string" default="/var/run//Dummy-default.state" />
```

Comment:
Too bad metadata output can't have underlining or italics. A quick look at man pages shows a variety of lowercase, UPPERCASE and <inbrackets>. Brackets would be really confusing in XML, and lowercase wouldn't indicate anything, so uppercase is probably best. I don't think curly braces are very clear; they do look like some code slipped through. Maybe "/var/run/Dummy-RESOURCE_ID.state" (in the metadata itself of course, as well as here).